usecases/auth/authentication/apikey/db_users.go

```diff
@@ -58,6 +59,7 @@ type User struct {
 
 type DBUser struct {
 	lock          *sync.RWMutex
+	weakHashLock  *sync.RWMutex
 	data          dbUserdata
 	memoryOnyData memoryOnlyData
```

Comment:
Typo. See [#7854](https://github.com/weaviate/weaviate/pull/7854)

Reply:
@mpartipilo approved the other PR